src/pivotal-ui/components/panels/panels.scss

```diff
@@ -130,175 +157,14 @@ Basic alternate panels are panels with a border and shadow. You can have a panel
 .panel-basic-alt {
   border: $panel-basic-alt-border;
   border-radius: $panel-basic-alt-border-radius;
+  overflow: hidden;
```

Comment:
Yeah, I don't know without strong user personas (or actual people) for those people.

It is _required_ though for rounding corners.